src/get-config.c

```diff
@@ -0,0 +1,44 @@
+/* 
+ * Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+ * Copyright (c) 2015-2017, Gregory M. Kurtzer. All rights reserved.
+ * Copyright (c) 2017, Vanessa Sochat All rights reserved.
+ * 
```

Comment:
ok added! I'm doing a few tweaks (waiting for @gmkurtzer to be able to take a look at something) and then I'll push the update.